mirage-runtime.opam

```diff
@@ -11,16 +11,17 @@ tags:         ["org:mirage" "org:xapi-project"]
 doc:          "https://mirage.github.io/mirage/"
 
 build: [
-  ["dune" "subst"] {pinned}
+  ["dune" "subst"] {dev}
   ["dune" "build" "-p" name "-j" jobs]
   ["dune" "runtest" "-p" name "-j" jobs] {with-test}
 ]
 
 depends: [
   "ocaml" {>= "4.08.0"}
-  "dune" {>= "2.0.0"}
+  "dune" {>= "2.8.0"}
   "ipaddr"             {>= "5.0.0"}
   "functoria-runtime"  {= version}
+  "functoria"          {= version & with-test}
```

Comment:
is this really needed? do we have mirage-runtime tests?

Reply:
I'm not sure indeed - do you prefer to delete (on both sides, `opam-repository` and this PR)?

Reply:
yes -- I can see a test/mirage-runtime directory, but there the test is attached (in the dune file) to the mirage package, not mirage-runtime